pkgs/top-level/all-packages.nix

```diff
@@ -19169,7 +19169,7 @@ with pkgs;
   openai-whisper = with python3.pkgs; toPythonApplication openai-whisper;
 
   openai-whisper-cpp = callPackage ../tools/audio/openai-whisper-cpp {
```

Comment:
```suggestion
  openai-whisper-cpp = darwin.apple_sdk_11_0.callPackage ../tools/audio/openai-whisper-cpp {
```

Reply:
Is this change also needed with the updated `darwin.apple_sdk_11_0.frameworks` below?

Reply:
Yes, so that frameworks are consistently on 11.0 on `x86_64-darwin`.